docs/UserGuide.adoc

```diff
@@ -598,19 +598,19 @@ Steps:
 1. Type the command with a category specified as shown in the screenshot below:
 
 .Screenshot of user input (category) into Command Box for Add Command in Cashier Tab
-image::casherUG/AddCommand1.png[]
+image::cashierUG/AddCommand1.png[]
```

Comment:
Can I know what was changed here?

Reply:
the folder name was misspelled so I changed it and all references to it